src/main/scala/nodes/stats/Sampling.scala

```diff
@@ -2,7 +2,7 @@ package nodes.stats
 
 import breeze.linalg.{DenseVector, DenseMatrix}
 import org.apache.spark.rdd.RDD
-import pipelines.{FunctionNode, Transformer}
+import pipelines.FunctionNode
```

Comment:
At the moment it's still implemented as a FunctionNode, but we can change it to a Transformer and not need FunctionNode.